{"name": "Accept with invalid order id returns client error", "status": "passed", "steps": [{"name": "Register and login courier to get courier_id", "status": "passed", "start": 1758377237490, "stop": 1758377237688}, {"name": "Attempt to accept clearly invalid order id", "status": "passed", "start": 1758377237688, "stop": 1758377237788}], "start": 1758377237490, "stop": 1758377237789, "uuid": "f40bba2a-357f-469a-b929-d5ed906d77fb", "historyId": "f44ffd4842848e56a1c086ca59e9a6ed", "testCaseId": "f44ffd4842848e56a1c086ca59e9a6ed", "fullName": "tests.test_accept_order#test_accept_order_with_invalid_order_id_returns_error", "labels": [{"name": "feature", "value": "Orders"}, {"name": "story", "value": "Accept order"}, {"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_accept_order"}, {"name": "host", "value": "MacBook-Air.local"}, {"name": "thread", "value": "74856-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_accept_order"}]}